roles/manage-user-access/tasks/main.yml

```diff
@@ -0,0 +1,22 @@
+---
+
+- include: 'authorizedkeys.yml'
+  when:
+  - user_name is defined
+  - authorized_keyfile is defined
+  - authorized_keyfile != ""
+
+- include: 'password.yml'
+  when: 
+  - user_name is defined
+  - user_name != ""
+  - user_name_password is defined
+  - user_name_password != ""
+
+- include: 'sshd-update.yml'
+  when: 
+  - update_sshd is defined
```

Comment:
The `update_sshd` is missing from the README documentation as far as I can tell?